src/pudl/transform/ferc1.py

```diff
@@ -1593,9 +1593,7 @@ def transform(
             )
 
     if "plants_steam_ferc1" in ferc1_settings.tables:
-        ferc1_transformed_dfs["plants_steam_ferc1"] = PlantsSteamFerc1(
-            table_name="plants_steam_ferc"
-        ).execute(
+        ferc1_transformed_dfs["plants_steam_ferc1"] = PlantsSteamFerc1().execute(
```

Comment:
I'm confused. This doesn't appear to be processing the fuel table at all, so how is the steam table okay? And it seems to be using your old transformer class, not the new one.  Am I just not understanding what the PR is meant to do?

Reply:
Okay after our phone chat I reverted that last commit that was an accidental mix of multiple branches. Now just the settings files are updated.